distribution/std-lib/Standard/src/Visualization/Helpers.enso

```diff
@@ -21,8 +56,8 @@ recover_errors ~body =
 
 ## PRIVATE
 
-   Returns all the columns in the table, including indices.
-   Index columns are placed before other columns.
+   Returns all the columns in the table, including indices. Index columns are 
+   placed before other columns.
```

Comment:
The summary line should be at most one sentence. Split the second sentence into another paragraph.